stock_prices/stock_prices.py

```diff
@@ -2,14 +2,25 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+  prices = prices.copy()
```

Comment:
I'm glad you listened to my suggestion and are no longer mutating the original list 😄